CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 - Added limited support for the `Timedelta` type, including
   - support `copy`, `cache_result`, `shift`, `sort_index`.
+  - support `astype` from `int` to `Timedelta`
```

Comment:
should be all numeric, like float and boolean too.

Reply:
Added support for float/other numeric types. Boolean to timedelta is not supported by native pandas either, which is also handled by our code now.

Reply:
Disregard previous comment, adding support for float + boolean.